packages/rancher-logging/overlay/templates/loggings/rke/daemonset.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.additionalLoggingSources.rke.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: "{{ .Release.Name }}-rke-aggregator"
+  namespace: "{{ .Release.Namespace }}"
+spec:
+  selector:
+    matchLabels:
+      name: {{ .Release.Name }}-rke-aggregator
+  template:
+    metadata:
+      name: "{{ .Release.Name }}-rke-aggregator"
+      namespace: "{{ .Release.Namespace }}"
+      labels:
+        name: {{ .Release.Name }}-rke-aggregator
+    spec:
+      containers:
+        - name: fluentd
+          image: "{{ template "system_default_registry" . }}{{ .Values.images.fluentbit.repository }}:{{ .Values.images.fluentbit.tag }}"
+          volumeMounts:
+            - mountPath: /var/lib/rancher/rke/log/
+              name: indir
+            - mountPath: /var/lib/rancher/logging/
+              name: outdir
+            - mountPath: /var/lib/docker/containers/
+              name: containers
+            - mountPath: /fluent-bit/etc/
+              name: config
+      volumes:
+        - name: indir
+          hostPath:
+            path: /var/lib/rancher/rke/log/
+        - name: outdir
+          hostPath:
+            path: /var/lib/rancher/logging/
```

Comment:
It's ok that both fluentD's output to same directory ?

Reply:
yes the files are different names